python/pyspark/sql/dataframe.py

```diff
@@ -697,6 +697,83 @@ def subtract(self, other):
         """
         return DataFrame(getattr(self._jdf, "except")(other._jdf), self.sql_ctx)
 
+    def dropna(self, how='any', thresh=None, subset=None):
```

Comment:
Either approach is okay as long as the APIs are consistent. We do have `groupBy`, `withColumn`, `toPandas`, `unionAll` and some others using camelCase. It is then a little inconsistent to use `dropna` and `fillna`.